src/test/kotlin/graphics/scenery/tests/unit/controls/SwingMouseAndKeyHandlerTests.kt

```diff
@@ -221,12 +221,12 @@ class SwingMouseAndKeyHandlerTests {
 
             val modifiers = listOf(
                 "",
-//                "ctrl",
+                "ctrl",
 //                "ctrl alt",
 //                "ctrl shift",
```

Comment:
Just curious: why do you test only one modifier at a time, instead of mod combos like these commented out ones?

Reply:
There were still some (I think platform-dependent) issues remaining with those. Once they are figured out, you're right, the other combos should also be tested 👍